Makefile

```diff
@@ -0,0 +1,24 @@
+# The following parameters are required:
+# - REPOSITORY
```

Comment:
Maybe a better name for this would be `GITHUB_URL`?

Reply:
refactor `RPOSITORY_URL` to be a bit agnostic? also, I believe this is set by `zero` so that would also change.

Reply:
Sure, though there's a bunch of other github stuff. "Repository" is fine, as long as we indicate that it should be a url.